docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,465 @@
+
+**DESCRIPTION**  
+Push the benchmark result to the Pbench Server without removing it from the local host. This command requires `/opt/pbench-agent/id_rsa` file with the private SSH key, when pushing to a 0.69 Pbench Server.
+
+**OPTIONS**  
+`--user`  
+This option value is required if not provided by the
+`PBENCH_USER` environment variable; otherwise, a value provided
+on the command line will override any value provided by the
+environment.
+
+`--controller`  
+This option may be used to override the value
+provided by the `PBENCH_CONTROLLER` environment variable; if
+neither value is available, the result of `hostname -f` is used.
+(If no value is available, the command will exit with an error.)
+
+`--prefix`  
+This option allows the user to specify an optional
+directory-path hierarchy to be used when displaying the result
+files on the 0.69 Pbench Server.
+
+`--show-server`  
+This will not move any results but will resolve and
+then display the pbench server destination for results.
+
+`--xz-single-threaded`  
+This will force the use of a single
+thread for locally compressing the result tar balls.
+
+`--help`  
+Show this message and exit.
+
+
+---
+
+### pbench-list-tools
+
+---
+
+**NAME**  
+`pbench-list-tools` - list all the registered tools optionally filtered by name or group
+
+**SYNOPSIS**  
+`pbench-list-tools [OPTIONS]`
+
+**DESCRIPTION**  
+List tool registrations, optionally filtered by tool name or tool group.
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`-n`, `--name` STR  
+List the tool groups in which tool <`STR`> is registered.
+
+`-g`, `--group` STR  
+List all the tools registered in the <`STR`> group.
+
+`-o`, `--with-option`  
+List the options with each tool
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-list-triggers
+
+---
+
+**NAME**  
+`pbench-list-triggers` - lists the registered triggers by group
+
+**SYNOPSIS**  
+`pbench-list-triggers[-C][-g][--help]`
+
+**DESCRIPTION**  
+This command will list all the registered triggers by `group-name`. 
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`-g`, `--group`, `--groups` STR  
+List all the triggers registered in the <`STR`> group.
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-move-results
+
+---
+
+**NAME**  
+`pbench-move-results` - move all results to 0.69 Pbench Server
+
+**SYNOPSIS**  
+`pbench-move-results [--help] [--user=<user>] [--controller=<controller>] [--prefix=<path>] [--xz-single-threaded] [--show-server]`
+
+**DESCRIPTION**  
+Push the benchmark result to the 0.69 Pbench Server , and requires a `/opt/pbench-agent/id_rsa` file with the private SSH key of the server's pbench account.
```

Comment:
On a successful push, this command removes the results from the local host, right?  We should mention that, here.